cluster/gce/gci/configure-helper.sh

```diff
@@ -425,6 +429,56 @@ function ensure-local-ssds() {
   done
 }
 
+function ensure-local-ssds-ephemeral-storage() {
+  # Get scsi devices
+  local devices=(/dev/disk/by-id/google-local-ssd-*)
+  if [ ! -e ${devices[0]} ]; then
+    echo "No local SCSI SSD disks found."
+    devices=()
+  fi
+  # Get nvme devices
+  for ssd in "/dev/nvme*"; do
+    if [ -e "${ssd}" ]; then
+      # This workaround to find if the NVMe device is a disk is required because
+      # the existing Google images does not expose NVMe devices in /dev/disk/by-id
+      if [[ `udevadm info --query=property --name=${ssd} | grep DEVTYPE | sed "s/DEVTYPE=//"` == "disk" ]]; then
+        devices+=("${ssd}")
+      fi
+    else
+      echo "No local NVMe SSD disks found."
+    fi
+  done
+  if [ "${#devices[@]}" -eq 0 ]; then
+    return
+  fi
+
+  # Create RAID array
+  local md_device=(/dev/md/*)
+  md_device=${md_device[0]}
+  if [ ! -e "$md_device" ]; then
+    md_device="/dev/md/0"
+    echo "y" | mdadm --create "${md_device}" --level=0 --raid-devices=${#devices[@]} ${devices[@]}
+  fi
+  local ephemeral_mountpoint="/mnt/disks/kube-ephemeral-ssd"
+  safe-format-and-mount "${md_device}" "${ephemeral_mountpoint}"
```

Comment:
Are the mount options and directory permissions what we want, such as the "a+w" permissions?

Reply:
yes, verified.